pkg/tsdb/cloudwatch/services/regions.go

```diff
@@ -32,10 +32,9 @@ func (r *RegionsService) GetRegions() ([]resources.ResourceResponse[resources.Re
 
 	result := make([]resources.ResourceResponse[resources.Region], 0)
 
-	ec2Regions, err := r.DescribeRegions(&ec2.DescribeRegionsInput{})
-	if err != nil {
```

Comment:
should we still log the error like the old code?

Reply:
An excellent point! Added!